extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -284,3 +285,72 @@
 .hidden {
 	display: none;
 }
+
+.cnw-select {
+	margin: $content-spacing-xxlarge 0;
+
+	.wds-dropdown {
+		border-bottom: 1px solid $wds-color-mist-gray;
+		width: $cnw-input-width-large;
+
+		.default-value {
+			line-height: $wds-line-height;
+		}
+
+		svg {
+			fill: $wds-color-light-gray;
+			float: right;
+			margin-top: 4px;
+		}
+
+		&.wds-is-active {
+			svg {
+				@include transform-rotate(180deg);
+			}
+		}
+
+		.wds-dropdown__content {
+
+			overflow: auto;
+			width: $cnw-input-width-large;
+
+			&.wds-dropdown__wiki-language {
```

Comment:
`.wds-` prefix suggests that this style comes from Wikia (Fandom) Design System—I would drop it.